docs/reference/connectors/clickhouse.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_label: elastic
+---
+
+# The `clickhouse` Connector
+
+The `clickhouse` collector aims integrate the [ClickHouse] database in Tremor. It has been tested with ClickHouse `v22.3-lts`.
```

Comment:
We need a two-sentence description here for folks who don't know clickhouse yet

Reply:
Wikipedia says: 
> ClickHouse is an open-source [column-oriented DBMS](https://en.wikipedia.org/wiki/Column-oriented_DBMS) (columnar database management system) for [online analytical processing](https://en.wikipedia.org/wiki/Online_analytical_processing) (OLAP)

Reply:
Thanks. This should be covered by [6c7d651](https://github.com/tremor-rs/tremor-www/pull/183/commits/6c7d65129081f618817ba4c4d312ba80d47aa534).